genesis/make_genesis_file.go

```diff
@@ -0,0 +1,199 @@
+package genesis
+
+import (
+	"bytes"
+	"encoding/csv"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"os"
+	"strconv"
+
+	ptypes "github.com/eris-ltd/eris-db/permission/types"
+	"github.com/eris-ltd/eris-db/util"
+
+	"github.com/tendermint/go-crypto"
+	wire "github.com/tendermint/go-wire"
+)
+
+//------------------------------------------------------------------------------------
+// core functions
+
+func GenerateKnown(chainID, accountsPathCSV, validatorsPathCSV string) (string, error) {
```

Comment:
GenerateKnown is currently the signature of the function as it is used by cli.  So my plan here is to rewrite package genesis fully, but keep `GenerateKnown` so that we can work on the content of genesis within eris-db without blocking work on cli.  Then afterwards the name and signature of that function can be swapped out.  `GenerateKnown`, in my mind, refers to "known accounts written in a csv file"